action_launcher.bash

```diff
@@ -22,11 +22,6 @@ ln -s "$(pwd)" "$recreated_repo_dir"
 
 cd "$recreated_repo_dir"
 
-if [ ! -f "$INPUT_CLANG_TIDY_FIXES" ]; then
-  echo "Could not find the clang-tidy fixes file '$INPUT_CLANG_TIDY_FIXES'. Perhaps it wasn't created?"
-  exit 0
-fi
-
```

Comment:
I'd say that it the fixes file is a _precondition_ for the Python script to run.

Therefore, my suggestion is to `touch` the fixes file if it's missing and print out the warning, instead of changing the logic of `run_action.py`.
What do you think?

Reply:
Hi @platisd to be honest, I would prefer to perform as few write operations as possible within this Action, especially if we are talking about files potentially provided by an untrusted user (and `fixes.yml` can be provided by an untrusted user if we analyze a pull request from a fork).

Reply:
Security concerns are rather low on my personal prio-list so it's always good to hear more opinions.
It sounds legit, so let's go with that.

Reply:
@platisd ZIP files may contain a symbolic links for example, and by issuing a `touch` on the symbolic link pointing to nowhere we will get a symbolic link pointing somewhere :)

Reply:
@platisd that's how it may look like:

```bash
$ ln -s nonexistent link
$ ls -l
total 0
lrwxrwxrwx 1 Oleg None 11 Feb  3 15:40 link -> nonexistent
$ [ -f link ] && echo exists
$ touch link
$ ls -l
total 0
lrwxrwxrwx 1 Oleg None 11 Feb  3 15:40 link -> nonexistent
-rw-r--r-- 1 Oleg None  0 Feb  3 15:40 nonexistent
$ [ -f link ] && echo exists
exists
```

`test -f` shows that there is no file, and it's sort of correct here, but there is a symlink and that's how it turns out :)